packages/visual-dev/src/components/RadioGeneric/RadioGeneric.stories.tsx

```diff
@@ -1,25 +1,25 @@
 import React from "react";
 import { useState } from "react";
-import { RadioGenericView, RadioOption } from ".";
+import { RadioGenericView } from ".";
 
 export default {
   title: "Components / Generic Radio",
   component: RadioGenericView,
 };
 
-const RadioHandle = ({
-  value,
-  activeValue,
-}: {
-  value: string;
-  activeValue: string;
-}) => {
-  return (
-    <div style={{ backgroundColor: value == activeValue ? "red" : "blue" }}>
-      {value}
-    </div>
-  );
-};
+// const RadioHandle = ({
+//   value,
+//   activeValue,
+// }: {
+//   value: string;
+//   activeValue: string;
+// }) => {
+//   return (
+//     <div style={{ backgroundColor: value == activeValue ? "red" : "blue" }}>
+//       {value}
+//     </div>
+//   );
+// };
```

Comment:
Flagging commented code

Reply:
This story needs to be fixed at some point, added a todo